src/Sarif/CodeGenHints.json

```diff
@@ -445,17 +612,35 @@
       }
     }
   ],
+  "ThreadFlowLocation.Importance": [
+    {
+      "kind": "EnumHint",
```

Comment:
>EnumHint [](start = 15, length = 8)

So this was just missing -- reveals test gap.

Reply:
Yep. Found this by updating the comprehensive.sarif example. We should really examine that thing more closely and continue to fill it out. very useful in many test scenarios.

---
In reply to: [221316544](https://github.com/Microsoft/sarif-sdk/pull/1103#discussion_r221316544) [](ancestors = 221316544)